packages/base-styles/package.json

```diff
@@ -24,6 +24,7 @@
 		"node": ">=18.12.0",
 		"npm": ">=8.19.2"
 	},
+	"wpScript": true,
```

Comment:
Is there any reason for adding this? If I recall correctly, this field is required for npm workspaces. See https://github.com/WordPress/gutenberg/pull/66272/files

Reply:
The reason why I added this was that it apparently gets used by the script that copies any dist files from the individual packages into the top level dist folder which is needed in order to properly enqueue this in the Gutenberg plugin.